UPGRADING

```diff
@@ -359,7 +364,7 @@ PHP 8.4 UPGRADE NOTES
 
 - Standard:
   . stream_bucket_make_writeable() and stream_bucket_new() will now return a
-    StreamBucket instance instead of an stdClass instance.
+    StreamBucket instance instead of a stdClass instance.
```

Comment:
```suggestion
    StreamBucket instance instead of an stdClass instance.
```

because it's pronounced as "es" so I think it needs to be "an".

Reply:
> because the s in stdClass pronounced as "es" so I think it needs to be "an".

Huh ? I'd be interested to hear how you pronounce it. In my mind, it's "standard class"...

Reply:
> because the s in stdClass pronounced as "es" so I think it needs to be "an".

hmm not sure I agree here :)

Reply:
Es tee dee class

Reply:
yeah I already read as standard class too.

Reply:
Maybe just avoid that ("instead of an instance of stdClass")?

Reply:
Correct is "an stdClass" because it's the similar rule as with `a URL` and similar examples.

Reply:
> Maybe just avoid that ("instead of an instance of stdClass")?

@cmb69 Let's ;-) Fixed up in the commit.

> Es tee dee class

@nielsdos Might just be me (and the fact that I did volunteer work for an AIDS foundation for over a decade), but pronouncing it as an acronym feels _unfortunate_ with the acronym being "std"... Unintended associations, I'm sure.